targets/XPlatCppSdk/source/cppsdk/include/playfab/PlayFabPluginManager.h

```diff
@@ -30,7 +30,8 @@ namespace PlayFab
     /// </summary>
     class IPlayFabHttpPlugin : public IPlayFabPlugin
     {
-        virtual void AddRequest(std::string url, std::unordered_map<std::string, std::string> headers, SharedVoidPointer callback, ErrorCallback errorCallback = nullptr, void* extrabuffer = nullptr);
+    public:
+        virtual void MakePostRequest(const std::string& url, const std::unordered_map<std::string, std::string>& headers, const std::string& requestBody, std::function<void(CallRequestContainerBase&)> callback, ErrorCallback errorCallback = nullptr, void* extrabuffer = nullptr) = 0;
```

Comment:
>std::string& url [](start = 43, length = 16)

Can you please consult this and check if we still need to pass strings by const reference ?
[https://stackoverflow.com/questions/10231349/are-the-days-of-passing-const-stdstring-as-a-parameter-over](https://stackoverflow.com/questions/10231349/are-the-days-of-passing-const-stdstring-as-a-parameter-over)